casl/mongoose 8.0.0 update, error with missing types

```
node_modules/.pnpm/@casl+mongoose@8.0.0_@casl+ability@6.7.1_mongoose@8.2.3/node_modules/@casl/mongoose/dist/types/accessibleBy.d.ts:2:32 - error TS2307: Cannot find module '@casl/ability/dist/types/types' or its corresponding type declarations.

import { ToAbilityTypes } from '@casl/ability/dist/types/types';
```

Hello, are there any ways to resolve this Typescript issue? Thanks!

--- a/packages/casl-mongoose/src/accessibleBy.ts
+++ b/packages/casl-mongoose/src/accessibleBy.ts
@@ -1,5 +1,4 @@
-import { AnyMongoAbility, Generics, SubjectType } from '@casl/ability';
-import { ToAbilityTypes } from '@casl/ability/dist/types/types';
+import { AnyMongoAbility, Generics, SubjectType, Abilities, AbilityTuple, ExtractSubjectType } from '@casl/ability';
 import { rulesToQuery } from '@casl/ability/extra';
 
 function convertToMongoQuery(rule: AnyMongoAbility['rules'][number]) {
@@ -23,12 +22,16 @@ export class AccessibleRecords<T extends SubjectType> {
   }
 }
 
+type SubjectTypes<T extends Abilities> = T extends AbilityTuple
+  ? ExtractSubjectType<T[1]>
+  : never;
+
 /**
  * Returns accessible records Mongo query per record type (i.e., entity type) based on provided Ability and action.
  */
 export function accessibleBy<T extends AnyMongoAbility>(
   ability: T,
   action: Parameters<T['rulesFor']>[0] = 'read'
-): AccessibleRecords<ToAbilityTypes<Generics<T>['abilities']>[1]> {
+): AccessibleRecords<SubjectTypes<Generics<T>['abilities']>> {
   return new AccessibleRecords(ability, action);
 }

